crypto/lhash/lhash.c

```diff
@@ -463,7 +463,7 @@ unsigned long lh_strhash(const char *c)
         v = n | (*c);
         n += 0x100;
         r = (int)((v >> 2) ^ v) & 0x0f;
-        ret = (ret << r) | (ret >> (32 - r));
+        ret = (ret << (r&31)) | (ret >> ((32-r)&31));
```

Comment:
Not protected by PEDANTIC and also not in master ...

Reply:
Nit: add space around "&" and "-"

Reply:
The line above does & 0x0f, there is little point in doing r&31.

Reply:
I also don't see the point in adding pedantic and non-pedantic version of this.